loops.js

```diff
@@ -18,6 +20,13 @@
 // klaar!
  // ==========================================
 
+for(let i = 0; i < 4; i++) {
+    console.log("Hoera!")
+}
+
+if (i=5) {
```

Comment:
Let op! 
= is een toekenningsoperator, het betekent "is"
=== is een vergelijkingsoperator, het betekent "is gelijk aan op waarde en type"

Hier zou je === moeten gebruiken in plaats van =